File: app/src/main/java/com/condorlabs/popularmovies/splash/SplashActivity.java
package com.condorlabs.popularmovies.splash;

import android.os.Bundle;
import android.os.Handler;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import com.condorlabs.popularmovies.home.MainActivity;

public class SplashActivity extends AppCompatActivity {

    private final static int SPLASH_DELAY = 2000;


    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        new Handler().postDelayed(() -> {
            finish();
            startActivity(MainActivity.newIntent(this));
        }, SPLASH_DELAY);
    }
}

File: app/src/main/java/com/condorlabs/popularmovies/splash/SplashActivity.kt
package com.condorlabs.popularmovies.splash

import android.os.Bundle
import android.os.Handler
import android.support.v7.app.AppCompatActivity

import com.condorlabs.popularmovies.home.MainActivity


class SplashActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Handler().postDelayed({
            finish()
            startActivity(MainActivity.newIntent(this))
        }, SPLASH_DELAY.toLong())
    }

    companion object {

        private const val SPLASH_DELAY = 2000
    }
}
